### PR TITLE
Improve webpage navbar and home-page

### PR DESCRIPTION
**Pull request by jorgepiloto**
As discussed privately, this tries to solve for https://github.com/poliastro/poliastro/issues/712, which was opened in order to have a better front page. 

At first, I created a for loop which iterated over all the different created pages. This was nice and simple but the problem came when linking those pages to the actual external content. This implementation enables maintainers to just modify the URLs in the actual `pelicanconfig.py` to redirect.

Some tasks still need to be solved:

- [x] Menu bar is completed
- [x] Home page with content
- [x] Show all posts in blog page

### CODE REVIEW

**Comment by jorgepiloto** [comment]
With the new update on blog location, last task is solved :heart_eyes:

**Comment by jorgepiloto** [comment]
The image below shows the looking this PR will impose on the actual web page. Notice that the `home` content is not completed, it was just a draft idea.

web

File: content/pages/home.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Welcome to poliastro webpage
```
Comment (astrojuanlu) [inline content/pages/home.rst:1]:
```suggestion
poliastro - Astrodynamics in Python
```

File: content/pages/home.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+============================
+
+:title: home
+:slug: home
+:template: page
+:index: 0
+:save_as: index.html
+:status: hidden
+
+A little brief description on the project and the website...?
```
Comment (astrojuanlu) [inline content/pages/home.rst]: Your suggestion presented in the screen capture looks OK for now!

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
This is great improvement over what we have now, and I am glad to merge it! After that we can:

* Address some infrastructure changes that I will detail in a separate issue
* Add more content here

Just left a couple of comments.

**Comment by shreyasbapat** [comment]
Can we have just a `pip install poliastro` over there?  Will look nice :p

**Comment by astrojuanlu** [comment]
Great catch @shreyasbapat !

**Comment by shreyasbapat** [comment]
@astrojuanlu Can we change our docs theme also?  :crying_cat_face:  

(I know the answer is no, but still :p)

File: content/pages/home.rst (hunk 1 above)
Comment (jorgepiloto) [reply content/pages/home.rst:1]: Solved :+1:

**Comment by astrojuanlu** [comment]
@shreyasbapat no :) sphinx-rtd-theme is, with all its problems, the best Sphinx theme available out there, nowadays. Sure, I'd like them to do more active maintenance... But I am sure they will pay more attention to it soon :)

**Comment by astrojuanlu** [comment]
@jorgepiloto What's the status of this? :)

**Comment by astrojuanlu** [comment]
Update: we fixed some local issues and this is almost ready to go 🚀

**Comment by jorgepiloto** [comment]
This is ready by my side @astrojuanlu!

**Comment by astrojuanlu** [comment]
This looks wonderful @jorgepiloto ! 😍 I'm so happy about this update, now we can promote this as the actual poliastro website rather than its documentation.

Merging and deploying the changes!

**Review by astrojuanlu — APPROVED** [review]

**Comment by jorgepiloto** [comment]
I'm glad you like it! It is already deployed and working as expected :rocket:

**Pull request merged**